paddle/c/scope.cc

```diff
@@ -0,0 +1,32 @@
+#include "scope.h"
```

Comment:
Pybind11 is also an excellent candidate https://media.readthedocs.org/pdf/pybind11/master/pybind11.pdf

Caffe2 chose pybind11 to expose `c++11` directly.